applenav/src/components/App.js

```diff
@@ -0,0 +1,14 @@
+import React, { Component } from 'react';
+import Header from './Header/Header';
+
+class App extends Component {
+  render() {
+    return (
+      <div>
+        <Header />
```

Comment:
You can set the mainbody component from here and have it set to it's corresponding subject.